src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Good use of access modifier, however, you could consider changing the variable name to a more descriptive one instead of 'by'